group_vars/all/vars.yml

```diff
@@ -45,6 +46,7 @@ sidekiq_netids:
   - eliotj # Eliot Jordan
   - hc8719 # Hector Correa
   - heberlei # Regine Herberlein
+  - is4684 # Isha Sinha
```

Comment:
It looks like this list is only used for pulmap and pulfalight, so I don't think you will need to be on this list.